go/vt/vtgate/planbuilder/update.go

```diff
@@ -112,8 +112,25 @@ func buildChangedVindexesValues(update *sqlparser.Update, table *vindexes.Table,
 	if len(changedVindexes) == 0 {
 		return nil, "", nil
 	}
+
+	var tableExpr sqlparser.TableExpr
```

Comment:
This may not be the most elegant approach. Initially I tried just printing out the entire TableExprs but that changed a bunch of query plans that use things like shard targeting to include the shard targeting in this owned vindex query.

I'm not sure if including shard targeting in the owned vindex query is ok, so I tried to avoid impacting any existing query plans.